codecov.yaml

```diff
@@ -30,12 +27,23 @@ coverage:
         # swinging coverage that is not triggered by changes in a PR.
         # See: https://twistedmatrix.com/trac/ticket/10170
         threshold: 0.02%
+      tests:
+        # Make sure we run each test at least once.
+        # If we don't have 100% coverage for tests, it meant that some
+        # tests are always skipped.
+        target: 100%
+        paths:
+          - "src/incremental/tests/*"
+          - "tests/*"
+          - "examplesetup.py"
```

Comment:
```suggestion
          - "tests/*"
```

Reply:
no examplesetup.py?

Reply:
I tried... but I was not able to enable the coverage for `examplesetup.py` ... it is only imported/executed  as `main`.